machines/settings.ts

```diff
@@ -205,3 +215,6 @@ export function selectCredentialRegistryResponse(state: State) {
 export function selectBiometricUnlockEnabled(state: State) {
   return state.context.isBiometricUnlockEnabled;
 }
+export function selectIsResetInjiProps(state: State) {
```

Comment:
we are exporting it and using it in controller but from controller we are not using it anywhere

Reply:
Actually is has to be used in settings screen to prevent user to click save while credential registery multiple times. Fixing it